django/db/backends/base/schema.py

```diff
@@ -200,6 +200,13 @@ def effective_default(self, field):
                 default = six.binary_type()
             else:
                 default = six.text_type()
+        elif getattr(field, 'auto_now', False) or getattr(field, 'auto_now_add', False):
+            default = timezone.now()  # default for DateTimeField
+            internal_type = field.get_internal_type()
+            if internal_type == 'DateField':
+                default = default.date()
```

Comment:
@aaugustin are there any reasons why DateField and TimeField are not using timezones while computing values for `auto_now`/`auto_now_add`? Seems there are inconsistency, which must be resolved, because values for effective_default must be the same as on the field implementation